src/Data/Skills/act_str.lua

```diff
@@ -7590,6 +7592,7 @@ skills["BloodstainedBanner"] = {
 	baseMods = {
 		skill("radius", 46),
 		skill("manaReservationPercent", 0, { type = "Condition", var = "BannerPlanted" }),
+		flag("Condition:AffectedByPlacedBanner", { type = "Condition", var = "BannerPlanted" }, { type = "GlobalEffect", effectType = "Buff" })
```

Comment:
Need to remember to add these lines to the act_str.txt file so they are exported from the game files. I'll add them this time

Reply:
Ah. Probably missed this in the Nightblade fix then as well. 🙈 

I'll fix that tomorrow.

Reply:
nvm, already fixed in [this commit](https://github.com/PathOfBuildingCommunity/PathOfBuilding/commit/591430285843171b7bece76723204d395cae5bd4#diff-c4d2f4bc4ef4a1fd2a1991410919c41890defce4960634338eaa8f4cc370c556R336). 👍